crates/re_renderer/src/allocator/uniform_buffer_fill.rs

```diff
@@ -75,12 +75,14 @@ pub fn create_and_fill_uniform_buffer_batch<T: bytemuck::Pod>(
         &ctx.gpu_resources.buffers,
         num_buffers as _,
     );
-    staging_buffer.extend(content);
-    staging_buffer.copy_to_buffer(
-        ctx.active_frame.before_view_builder_encoder.lock().get(),
-        &buffer,
-        0,
-    );
+    staging_buffer.extend(content).ok(); // We just allocated the buffer for this exact size.
```

Comment:
I rather we just do `unwrap` to be honest - at least we'll find the bug then instead of having it hidden. Or call a `log_on_error`